kroxylicious-runtime/src/main/java/io/kroxylicious/proxy/config/Feature.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious.proxy.config;
```

Comment:
I think this amounts to a public API if its in this package. Maybe it should live in `io.kroxylicious.proxy.internal.config` or something similar?